src/ui/components/Header/Header.react.js

```diff
@@ -102,7 +96,7 @@ const mapStateToProps = ({ player }) => ({
   shuffle: player.shuffle,
   queue: player.queue,
   queueCursor: player.queueCursor,
-  useNativeFrame: config.get('useNativeFrame'),
+  showTopHeader: os.platform() !== 'darwin' && !config.get('useNativeFrame'),
```

Comment:
can't we group these together to get rid of negations? Like:
```js
showTopHeader: !(os.platform() === 'darwin' && config.get('useNativeFrame'))
```

Reply:
I think the correct condition would be `!(os.platform() === 'darwin' || config.get('useNativeFrame'))`, which I find quite less understandable than the current one

Reply:
I will add a comment though

Reply:
Can we extract all those platform related conditions somewhere specific for the whole application ?
This would remove all those messy conditions `!os.specific && foobar`

Reply:
Yeah we might want that at some point indeed